hibernate-core/src/main/java/org/hibernate/query/hql/internal/SqmPathRegistryImpl.java

```diff
@@ -170,7 +170,7 @@ public SqmFrom findFromExposing(String navigableName) {
 		for ( SqmFrom fromElement : sqmFromByPath.values() ) {
 			if ( definesAttribute( fromElement.getReferencedPathSource(), navigableName ) ) {
 				if ( found != null ) {
-					throw new IllegalStateException( "Multiple from-elements expose unqualified attribute : " + navigableName );
+//					throw new IllegalStateException( "Multiple from-elements expose unqualified attribute : " + navigableName );
```

Comment:
@sebersole I had to remove this exception in order to make the following query work:

    insert into Foo (bar) select f.bar from Foo f

However, I feel like it should not have been necessary to make this change, and that there is some deeper problem here. The subquery should have its own inner-scoped namespace.

Reply:
Not sure.  Ideally add some *specific* tests for scoping and see.